config/e2e/diagnostics_job.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .PodName }}
+  namespace: {{ .ESNamespace }}
+  labels:
+    test-run: {{ .Context.TestRun }}
+spec:
+  terminationGracePeriodSeconds: 0
+  initContainers:
+    - name: diagnostics
+      # image built as described in https://github.com/elastic/support-diagnostics#creating-a-docker-image
+      image: docker.elastic.co/eck-dev/support-diagnostics:412ac7a392cf
+      imagePullPolicy: IfNotPresent
+      env:
+      - name: ES_PW
+        valueFrom:
+          secretKeyRef:
+            name: {{ .ESName }}-es-elastic-user
+            key: elastic
+      command: [sh, -c]
+      args:
+      - ./diagnostics.sh -h {{.ESName}}-es-http --type api --bypassDiagVerify -u elastic --passwordText $ES_PW {{if .TLS}}-s --noVerify{{end}} -o /diagnostic-output
+      volumeMounts:
+      - name: output
+        mountPath: /diagnostic-output
+  containers:
+  - name: offer-output
+    image: alpine
```

Comment:
Would it be possible to rely on the same Docker image as the one used for the `diagnostics` container ?
The purpose would be to avoid to add an additional dependency on an external infra. Also I don't know how this image is maintained.

Reply:
Yes definitely. I changed it.